diff --git a/src/Controller/OvenController.php b/src/Controller/OvenController.php
new file mode 100644
index 0000000..cd6cabe
--- /dev/null
+++ b/src/Controller/OvenController.php
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Pizza;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+class OvenController
+{
+    /**
+     * @Route("/")
+     */
+    public function cook(ValidatorInterface $validator)
+    {
+        $pizza = new Pizza();
+
+        $validator->validate($pizza);
+
+        return new Response('will not get here');
+    }
+}
